src/views/tables/PositionsTable.tsx

```diff
@@ -335,7 +335,9 @@ const getPositionsTableColumnDef = ({
           key:
             shouldRenderTriggers && showClosePositionAction && !testFlags.isolatedMargin
               ? STRING_KEYS.ACTIONS
-              : STRING_KEYS.CLOSE,
+              : showClosePositionAction
+              ? STRING_KEYS.CLOSE
+              : STRING_KEYS.ACTION,
```

Comment:
[nit] can we define this variable somewhere else/abstract this to a function? this ternary is getting pretty long and a little difficult to read

Reply:
leaving this as is for now -- will follow up when we clean up the `shouldRenderTriggers` variable which should help